### DISCUSSION TITLE
SAJ eSolar

### DISCUSSION THREAD

**Discussion by svengebe** [faanskit/ha-esolar#11] [Q&A]
Hoping someone can help me out, I am trying to log in but with no success, I did check my credentials are correct logging in via the normal browser, I know when you log in you have to select region, maybe its set wrong, just not sure how to fix it, very new HA.  
![Screenshot 2024-07-20 163813](https://github.com/user-attachments/assets/79d44876-7754-4e79-9a21-0c767339703a)

**Comment by faanskit** [comment]
Same issue as #10?
Use https://github.com/djansen1987/SAJeSolar instead

**Comment by svengebe** [comment]
Yea unfortunately
Thanks!

I’ll give that a shot.

Regards
Sven Gebert

**Comment by slippery-carrot** [comment]
Just had an R5 installed with an AOI3 wifi adaptor. Fker of a sales rep told me it'd work no drama with HA. He lied. 

Anyway, if you download the integration from HACS, make sure you change which release you want to download, mine defaulted to 0.9 not 0.10. 

0.10 has the region switch added, which solved my issues. You're all legends, speaking to the kings and/or queens that worked on this integration.

**Comment by faanskit** [comment]
Version 0.10.0 is a pre-release and never tested after merge. Happy it worked. I can now make it official.

  
  Very little time now goes to this project as I no longer have a SAJ inverter. But there exist another HA integration and there are a few competent developers using it and submitting PR's.
  
  
    
  
  /Marcus
    
    
    
    ------ Original Message ------
       Received: 07:11 AM CET, 11/27/2024

**Reply by slippery-carrot** [reply]
Yea copy. I tried the other one but it wouldn't work, it didn't seem to take into account the different URLs based on region. I tried changing it in the integration code but it didn't work.
I'm not very familiar with python, and writing integrations for HA, and I don't have the time at the moment to learn it. 
I was really hoping for a simple set-up solution like most integrations. 

I'll be focusing my time tomorrow getting the integration to work with the energy management side of HA.

Cheers again :)

**Reply by lucianor** [reply]
can you please let me know which version did you use? a PR has been closed that added the region switch:
https://github.com/faanskit/ha-esolar/pull/12
And has since been released:
https://github.com/faanskit/ha-esolar/releases/tag/V0.0.10

Did you try with this version?

**Reply by slippery-carrot** [reply]
Yup! That's the one :)

Dunno if it's helpful to know, looks like they're updating to a new website, there's a button to check out the new website. All same info just a new UI

Redirects to https://iop.saj-electric.com/index
